### DISCUSSION TITLE
Bulk publishing to the confluence.

### DISCUSSION THREAD

**Discussion by z0lberg** [docToolchain/docToolchain#826] [Q&A]
Hello everyone! Could you please help me with publishing to the confluence? Is there a way to use some mask instead of the concrete file name? 
![image](https://user-images.githubusercontent.com/10826294/158984714-0bdaa20b-858a-4bf7-8c10-3eb3314486c0.png)

**Comment by rdmueller** [comment]
Currently, a mask is not implemented. The standard use case is to create one large html file which includes all docs you want to publish.

This large file is is usually created from an `.adoc` file which includes other files.

What would be your use case for a mask?

**Reply by z0lberg** [reply]
Hi! We are having several repo with different organization styles inside. So I have to provide some common solution for getting docs to the our confluence. I can add mask support, do you have any suggestions or vises here?

**Reply by rdmueller** [reply]
If you can add it, feel free to suggest a style. Could also make sense for the `generateHTML` task :-)

**Reply by z0lberg** [reply]
Hi! I prepared changes for bulk publishing to the confluence, for the generateHTML task looks already implemented . The backward compatibility is keeped. 
Also i found one more place to improve: there is some misleading behavior with ancestorName and ancestorId - ancestorName is not available for the global context. I want to make those in a same way.

One more thing - for bulk publishing I might to create something like using file name convention for specifying an ancestorName from the file name. Functionality like this might be no need for common project.

**Reply by rdmueller** [reply]
great. I've just merged your PR.

file name conventions and idea with ancestorName/Id also sounds like a good idea to me...
